aten/src/ATen/native/Math.h

```diff
@@ -220,16 +220,9 @@ static inline double zeta(double x, double q) {
   return s;
 }
 
-static inline double polevl(double x, double *A, size_t len) {
-  double result = 0;
-  for (size_t i = 0; i <= len; i++) {
-    result = result * x + A[i];
-  }
-  return result;
-}
-
-static inline float polevlf(float x, float *A, size_t len) {
-  float result = 0;
+template <typename T>
```

Comment:
Would you add a comment for this function? How to use it and what it's intended for?